### PR TITLE
Add support for ModelValidation

### PR DESCRIPTION
**Pull request by apdavison**

### CODE REVIEW

File: schemas/validationTest.schema.tpl.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        "uniqueItems": true,
+        "_instruction": "Add one or several versions of this validation test.",
+        "_linkedTypes": [
+          "https://openminds.ebrains.eu/computation/ValidationTestVersion"
+        ]
+      },
+      "scope": {
+        "_instruction": "Add the scope of this validation test.",
+        "_linkedTypes": [
+          "https://openminds.ebrains.eu/controlledTerms/ModelScope"
```
Comment (lzehl) [inline schemas/validationTest.schema.tpl.json:35]: I would have expected here a different set of terms. I would have assumed that the ModelScope is for example "single cell" and the ValidationTestScope refers to the scope of the test not the model it is testing.

File: schemas/modelValidation.schema.tpl.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+    "_type": "https://openminds.ebrains.eu/computation/ModelValidation",
```
Comment (lzehl) [inline schemas/modelValidation.schema.tpl.json:2]: where is this activity used? how does this connect to the validation test (version) schemas?

**Review by lzehl — COMMENTED** [review]
@apdavison I have only a few questions that need potentially just clarification.

File: schemas/validationTest.schema.tpl.json (hunk 1 above)
Comment (apdavison) [reply schemas/validationTest.schema.tpl.json:35]: It is supposed to be the same set of terms for models and tests. The usage is, for example, "show me all validation tests that are potentially relevant for my single cell model".

File: schemas/modelValidation.schema.tpl.json (hunk 2 above)
Comment (apdavison) [reply schemas/modelValidation.schema.tpl.json:2]: It is used in workflows. The connection is in the "input" property, which includes ValidationTestVersion.

File: schemas/validationTest.schema.tpl.json (hunk 1 above)
Comment (lzehl) [reply schemas/validationTest.schema.tpl.json:35]: okay

File: schemas/modelValidation.schema.tpl.json (hunk 2 above)
Comment (lzehl) [reply schemas/modelValidation.schema.tpl.json:2]: okay

**Comment by lzehl** [comment]
@apdavison feel free to merge! Thanks for the clarifications

**Pull request merged**